docker-inasafe/Dockerfile

```diff
@@ -0,0 +1,29 @@
+#--------- Generic stuff all our Dockerfiles should start with so we get caching ------------
+FROM ubuntu:14.04
+MAINTAINER Akbar Gumbira <akbargumbira@gmail.com>
+
+RUN export DEBIAN_FRONTEND=noninteractive
+ENV DEBIAN_FRONTEND noninteractive
+RUN dpkg-divert --local --rename --add /sbin/initctl
+
+# Use local cached debs from host (saves your bandwidth!)
+# Change ip below to that of your apt-cacher-ng host
+# Or comment this line out if you do not wish to use caching
+ADD 71-apt-cacher-ng /etc/apt/apt.conf.d/71-apt-cacher-ng
+
+RUN echo "deb http://archive.ubuntu.com/ubuntu trusty main universe" > /etc/apt/sources.list
+RUN apt-get -y update
+RUN apt-get -y install rpl pwgen git
+
+#-------------Application Specific Stuff ----------------------------------------------------
+# Install QGIS 2.4, git, xvfb, and paramiko
+RUN echo "deb http://qgis.org/debian trusty main" >> /etc/apt/sources.list; echo "deb-src http://qgis.org/debian trusty main" >> /etc/apt/sources.list
+RUN gpg --keyserver keyserver.ubuntu.com --recv DD45F6C3; gpg --export --armor DD45F6C3 | sudo apt-key add -
+RUN apt-get -y update
+RUN apt-get -y install qgis python-qgis git python-paramiko xvfb
+ADD qgis-2.4 /usr/local/qgis-2.4
```

Comment:
This looks old - we should be basing off the kartoza/qgis-desktop image

Let me look and see if there is some uncommitted stuff in my tree here...